wsdiscovery/util.py

```diff
@@ -259,14 +261,14 @@ def _getNetworkAddrs(protocol_version):
             for ip in iface.ips:
                 if isinstance(ip.ip, str):
                     ip_address = ipaddress.ip_address(ip.ip)
-                    if not ip_address.is_loopback:
+                    if not ip_address.is_loopback and ip_address not in _APIPA_RANGE:
```

Comment:
You could probably use `is_link_local` instead https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address.is_link_local

Reply:
Ah, yeah that would probably work. Looks like the PR was already merged, but I can test it out and send another PR to simplify if it works.

Reply:
yep, it works: sent https://github.com/andreikop/python-ws-discovery/pull/81